spec/services/solidus_bolt/users/sync_payment_sources_service_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe SolidusBolt::Users::SyncPaymentSourcesService, :vcr, :bolt_configuration do
+  subject(:sync_payment_sources) do
+    described_class.call(user: user, access_token: ENV.fetch('BOLT_ACCESS_TOKEN', 'Fake_access_token'))
+  end
+
+  let(:user) { create(:user) }
+
+  describe '#call' do
+    it 'creates a new payment source with card ID' do
+      expect { sync_payment_sources }.to change { SolidusBolt::PaymentSource.count }.by(1)
+      bolt_payment_source = SolidusBolt::PaymentSource.last
+      expect(bolt_payment_source.card_id).to be_truthy
+      expect(bolt_payment_source.card_last4).to be_truthy
+      expect(bolt_payment_source.card_expiration).to be_truthy
```

Comment:
`be_truthy` is used usually to check whether the value is not false or nil. Considering here we just have to check that the value is not nil, I think be_present would be better.

Reply:
fixed in c1d7b19